api/controllers/v0.3/AddressValidationAPI.js

```diff
@@ -30,7 +28,7 @@ const AddressValidationAPI = (args = {}) => {
   };
 
   /**
-   * validate(address)
+   * validate
```

Comment:
Why not keep the arguments in the comments?  Makes it easier to understand what the function does...

Reply:
I felt it was redundant since the comment has the `@param` example and then the function definition is right below it. I'm not sure what the "right" format is; I've seen both examples in multiple repos but without the param seems more common.